packages/patternfly-4/react-charts/src/components/ChartBar/examples/ChartBar.md

```diff
@@ -2,28 +2,33 @@
 title: 'Bar'
 section: 'charts'
 typescript: true
-propComponents: ['Chart', 'ChartAxis', 'ChartBar', 'ChartGroup', 'ChartVoronoiContainer']
+propComponents: [
+  'Chart',
+  'ChartAxis',
+  'ChartBar',
+  'ChartGroup',
+  'ChartVoronoiContainer'
+]
 ---
 
 import { Chart, ChartAxis, ChartBar, ChartGroup, ChartThemeColor, ChartVoronoiContainer } from '@patternfly/react-charts';
-import './chart-bar.scss';
 
+## Introduction
 Note: PatternFly React charts live in its own package at [@patternfly/react-charts](https://www.npmjs.com/package/@patternfly/react-charts)!
 
-## Introduction
 PatternFly React charts are based on the [Victory](https://formidable.com/open-source/victory/docs/victory-chart/) chart library, along with additional functionality, custom components, and theming for PatternFly. This provides a collection of React based components you can use to build PatternFly patterns with consistent markup, styling, and behavior.
 
 Learn to build a bar chart using a Katacoda tutorial starting with a simple chart, adding multiple datasets, tooltips, axis labels, a legend, and concluding by changing the theme color. You'll learn how to use React chart components together to build a consistent user experience.
 
 [Start course](https://katacoda.com/patternfly/courses/charts/bar-chart)
 
 ## Examples
-```js title=Simple-bar-chart-with-right-aligned-legend
+```js title=Right--aligned-legend
```

Comment:
We intentionally used "Simple bar chart" here. Can we please put that title back in place?

Also, "right aligned" doesn't need the dash. We only did that for 'multi-color' and 'bottom-left', since that matches the prop values.